nixos/doc/manual/configuration/customizing-packages.section.md

```diff
@@ -1,11 +1,23 @@
 # Customising Packages {#sec-customising-packages}
 
-Some packages in Nixpkgs have options to enable or disable optional
-functionality or change other aspects of the package.
+The Nixpkgs configuration for a NixOS system is set by the {option}`nixpkgs.config` option.
+
+:::{.example}
+# Globally allow unfree packages
```

Comment:
Since this is now an example and is a self-contained "block", might also be a good idea to move that sentence ("However, this does not allow unfree software...") here after a slight adjustment to make it more understandable.